src/background/commands/alias.js

```diff
@@ -0,0 +1,27 @@
+/* @flow strict */
+
+import type { StoreState } from 'types';
+
+/* This command has essentially the same logic as env.js */
+export default function alias(state: StoreState, params: Array<string>) {
+  if (!state.wsh.aliases) {
+      state.wsh.aliases = {};
+  }
```

Comment:
We could add `aliases` to the initial state (just initialize to `{}`) to avoid this null check

Reply:
As discussed, won't solve the issue of upgrading systems; I think it's best for commands not to depend heavily on what the state might be and resolve as needed.